registry.json

```diff
@@ -1,4 +1,15 @@
 [
+{
+    "ticker": "AMDLO",                                                       
+    "poolId": "2730ca79a60b3b3f29b709bd39c21d66cb2c83e5441c479f755d0a3d", 
+    "social": {       
+      "website": "https://armadillo-pool.org",
+      "twitter": "Armadillo_Pool", 
+      "instagram": "",
+          },
+    "telegram": "Zebra [AMDLO]",                                                                           
```

Comment:
Please make this the username of an individual and not a group or channel. This looks like a screenname rather than username